src/main/java/bio/terra/cli/command/resource/create/BqDataset.java

```diff
@@ -17,6 +18,7 @@
     showDefaultValues = true)
 public class BqDataset extends BaseCommand {
   @CommandLine.Mixin ControlledResourceCreation controlledResourceCreationOptions;
+  @CommandLine.Mixin BqDatasetLifetime bqDatasetLifetime;
```

Comment:
nit: can you use the `Options` suffix for the variable name to be consistent with the mixin above?